src/sass/components/_post-box.scss

```diff
@@ -2,16 +2,28 @@
   margin-bottom: 2rem;
 
   .cover {
-    padding: 70% 25px 0 25px;
+    position: relative;
+    padding: 0px;
```

Comment:
0 to 0, nie ma jednostki

Reply:
heh oczywiście. nieuwaga - pozostałość po tym, co było wcześniej. już poprawiam i robię pull request.